apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/FilterPartial.cs

```diff
@@ -76,19 +76,25 @@ public static Filter And(IEnumerable<Filter> filters)
         /// Creates a filter to check that the specified property is in a given array of values.
         /// </summary>
         /// <param name="value">The name of the property. Must not be null.</param>
-        /// <param name="collection">The array of values to compare against. </param>
+        /// <param name="collection">The array of values to compare against. Must not be null. </param>
         /// <returns>The newly created filter.</returns>
-        public static Filter In(string value, ArrayValue collection) =>
-            Property(value, collection, Operator.In);
+        public static Filter In(string value, ArrayValue collection)
+        {
+            GaxPreconditions.CheckArgument(collection != null, nameof(collection), "Filter collection must not be null");
```

Comment:
Please use `GaxPreconditions.CheckNotNull(collection, nameof(collection))` here and below.